### DISCUSSION TITLE
dnf5 repoquery -f short option not recognized

### DISCUSSION THREAD

**Discussion by yrro** [rpm-software-management/dnf5#1827] [Q&A] [answered]
Hi folks, is there a reason that `dnf5 rq -f ...` doesn't work? The functionality is still available with the long `--files=` option.

I can file an issue if this is not already known about.

(And congrats on dnf5 becoming the default in Fedora 41 - it's a lovely improvement upon dnf4 in terms of both speed and command line interface consistency!)

**Comment by kontura** [comment, marked as answer]
I have actually forgot to add also other short options (`-i` and `-s`).
The issue would be great.

**Reply by yrro** [reply]
Thanks - I have filed #1835.
